packages/twenty-front/src/modules/object-record/hooks/__tests__/useCreateManyRecords.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { ReactNode } from 'react';
+import { MockedProvider } from '@apollo/client/testing';
+import { act, renderHook } from '@testing-library/react';
+import { RecoilRoot } from 'recoil';
+
+import {
+  query,
+  responseData,
+  variables,
+} from '@/object-record/hooks/__mocks__/useCreateManyRecords';
+import { useCreateManyRecords } from '@/object-record/hooks/useCreateManyRecords';
+
+const people = [
+  { id: 'a7286b9a-c039-4a89-9567-2dfa7953cda9' },
+  { id: '37faabcd-cb39-4a0a-8618-7e3fda9afca0' },
+];
+
+const mocks = [
+  {
+    request: {
+      query,
+      variables,
+    },
+    result: jest.fn(() => ({
+      data: {
+        createPeople: people.map((person) => ({
+          id: person.id,
+          ...responseData,
+        })),
+      },
+    })),
+  },
+];
+
+const Wrapper = ({ children }: { children: ReactNode }) => (
+  <RecoilRoot>
+    <MockedProvider mocks={mocks} addTypename={false}>
```

Comment:
@gitstart-twenty does this fail if the request does not match the mock?

Reply:
The test ideally won't fail unless we're specifically asserting on the request or response data. However, Apollo will issue warnings if a request is made and not properly mocked.